src/chains/ethereum/ethereum/src/forking/persistent-cache/persistent-cache.ts

```diff
@@ -1,6 +1,9 @@
 import { Tree } from "./tree";
```

Comment:
Most of these changes are just reverting back to what we did in `develop`. The only real change is using the `GanacheLevelUp` type instead of `LevelUp`